### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by ChinatuL** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#43] [Outreachy 2022] [answered]
Hi, My name is Chinatu Lucia from Nigeria. I am one of the outreachy interns and I would love to contribute to the projects for this community.

**Comment by BatoolMM** [comment, marked as answer]
Welcome, @ChinatuL to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
